### ISSUE TITLE
getting a decode error for the result response 

### ISSUE THREAD

**Issue by chetan-diy-ios** [adamrushy/OpenAISwift#11]
Hi Team, 

I tried integrating the package and facing a decoding error in the response. Could some one please guide me 
my code 

`func getData(text: String, completion: @escaping ((Result<String, Error>) -> Void)) {

        client?.sendCompletion(with: text) { result in
            switch result {
            case .success(let success):
                if let atleastOneReply = success.choices.first?.text {
                    completion(.success(atleastOneReply))
                }
            case .failure(let failure):
                print(failure.localizedDescription)
            }
        }
    }`

<img width="940" alt="Screenshot 2022-12-20 at 4 19 14 PM" src="https://user-images.githubusercontent.com/98536804/208665864-96a8d4f2-d2b9-43e9-af32-76bb50db8a85.png">

**Comment by adamrushy** [comment]
Hey, @chetan-diy-ios 

Can you share the `text` you're sending, please?

Thank you,
Adam

**Comment by chetan-diy-ios** [comment]
I tried many questions.
"Hello how are you"
"Biggest animal on planet"
"when is iPhone will actually get a update"

**Comment by andrewcar** [comment]
I am getting the same issue.

**Comment by rishav4336** [comment]
@andrewcar which characters are you talking about? I am also facing same error. Maybe your solution can help.

**Comment by adamrushy** [comment]
Thank you @andrewcar for confirming. So let me share my sample code;

```
        openAI.sendCompletion(with: "Hello how are you", maxTokens: 100) { result in
            switch result {
            case .success(let success):
                print(success.choices.first?.text ?? "")
            case .failure(let failure):
                print(failure.localizedDescription)
            }
        }
```

Console output is:

```
I'm doing great, thank you! How are you?
```

**Comment by andrewcar** [comment]
> @andrewcar which characters are you talking about? I am also facing same error. Maybe your solution can help.

If you go to [your API keys](https://beta.openai.com/account/api-keys), and if you have generated any secret keys, you'll see them there with the middle truncated. 

So, when I visited there to double check mine, I noticed that the last characters didn't match up with what I was using to initialize the Swift library. I just created a new one and my issue was resolved.
